### ISSUE TITLE
ZX300 support

### ISSUE THREAD

**Issue by unknown321** [unknown321/wampy#12]
Need a device to test on or a volunteer. Volunteer should make a working backup of their device (or at least try to!). How to make a backup - https://github.com/unknown321/wampy/blob/master/BACKUP.md

**Comment by unknown321** [comment]
Your currently installed firmware is **stock** and you are installing wampy for **walkman one**. What did you expect to happen? Perhaps my instructions were off somewhere?

You can try this build. Select "ZX300" during installation. https://github.com/unknown321/wampy/releases/tag/zx300stock

**Comment by Klonoa197** [comment]
Hello! My current firmware is the stock version, and I installed Wampy to use the Wampy Config Editor. However, your playbook doesn’t include a setup guide for the Editor page.

**Comment by unknown321** [comment]
> Hello! My current firmware is the stock version, and I installed Wampy to use the Wampy Config Editor. However, your playbook doesn’t include a setup guide for the Editor page.

>Wampy Config Editor.

What's that? Do you mean Walkman One config editor like on [image here](https://github.com/unknown321/wampy/blob/master/USAGE.md#walkman-one)? Or are you talking about [volume table editor here](https://github.com/unknown321/wampy/blob/master/SOUND_SETTINGS.md#master-volume)?

**Comment by Klonoa197** [comment]
Yes! That’s exactly it. I followed your playbook step by step, but it doesn’t seem to work. The built-in storage of the device doesn’t show any config files or related settings.

**Comment by unknown321** [comment]
That means you installed wrong version. Is there upgrade.log file on internal storage?

**Comment by Klonoa197** [comment]
NO, I downloaded at [ wampy.2025-02-03.b6cec47.exe ](https://github.com/unknown321/wampy/releases/download/v1.6.3/wampy.2025-02-03.b6cec47.exe) and installed it, but there’s no log file at all. Also, the [firmware you provided earlier](https://github.com/unknown321/wampy/releases/tag/zx300stock) —was it the official firmware or a modified CFW based on the official version?

**Comment by unknown321** [comment]
It was a model mismatch so nothing was installed.
